.github/workflows/codeql.yml

```diff
@@ -26,6 +26,74 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: apt dependencies
+        run: |
+          sudo apt-get --assume-yes update
+          sudo apt-get --assume-yes install build-essential git g++ autotools-dev libxml-parser-perl wget meson mm-common libgtk-4-bin libgtk-4-common libgtk-4-dev libgtk-4-doc pkg-config
+
+      - name: Build and install libsigc++
```

Comment:
Any reason why you're compiling these dependencies from source? Isn't gtkmm4 installable through `apt-get`?

Reply:
That's fine then, they are indeed currently using jammy for ubuntu-latest according to their docs.

This is somewhat surprising though given that the jammy CI run is succeeding, but closer inspection reveals that the tests are not running at all.

I think that [here ](https://github.com/tschoonj/gtkmm-plplot/pull/52/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR156) we need to replace `seperator` with `separator`.

I am fine replacing jammy with kinetic as base docker image though, as I wouldn't like to install those dependencies from source.

Reply:
I am fine with moving to fedora, probably best to use `fedora:latest`, and similar for Ubuntu: `ubuntu:latest`, which should map to `lunar` at the moment

Reply:
We could use `ubuntu:rolling` then perhaps?

Reply:
Thanks, looks good. I will try and review the PR properly over the weekend, when I have a bit more time.